scripts/get_famous_people_list/src/get_famous_people_list.py

```diff
@@ -0,0 +1,63 @@
+import requests
+import spacy
+from bs4 import BeautifulSoup
+
+MAIN_WIKI = 'https://en.wikipedia.org/wiki/Lists_of_people_by_nationality'
+BASE_WIKI = 'https://en.wikipedia.org'
+NLP_MODEL = spacy.load('en_core_web_lg')
+
+
+def get_pages(demonym_file):
+    """Gets individual famous people pages by nationality
+    
+    Arguments:
+        demonym_file {txt} -- path to file containing country demonyms
+    
+    Returns:
+        str[] -- list of urls to individual country pages
+    """
+
+    result = requests.get(MAIN_WIKI)
+    soup = BeautifulSoup(result.content, 'html.parser')
+
+    demonyms = []
+    with open(demonym_file) as fp:
+        demonyms = fp.read().splitlines() 
+
+    pages = []
+    first_ul = soup.find_all('ul')[1]
+    for li_tag in first_ul.find_all('li'):
+        if li_tag.text in demonyms:
+            pages.append(BASE_WIKI + li_tag.a.attrs['href'])
+    return pages
+
+
+def get_people(pages, output_file):
+    """Given a list of pages, gets all people's names
+    
+    Arguments:
+        pages {str[]} -- list of urls to famous people from country pages
```

Comment:
Nit: instead of comments, consider using Python 3 type annotations which tend to get maintained more than comments. E.g.:

```
from typing import List

def get_people(pages: List[str], output_file: str):
```

Reply:
fixed!